ngraph/src/ngraph/frontend/onnx_import/op/lstm.cpp

```diff
@@ -239,9 +243,17 @@ namespace ngraph
                         attributes.m_activations,
                         attributes.m_clip_threshold,
                         attributes.m_input_forget);
-                    return {std::make_shared<ngraph::opset0::GetOutputElement>(lstmSequence, 0),
-                            std::make_shared<ngraph::opset0::GetOutputElement>(lstmSequence, 1),
-                            std::make_shared<ngraph::opset0::GetOutputElement>(lstmSequence, 2)};
+
+                    const auto Y =
```

Comment:
I may be not really familiar with nGraph part. But still. It feels like GOE is not allowed to use, does it? @ilyachur, @GlebKazantaev please comment.

Reply:
Yes, Scott made a big work in order to remove GOE from nGraph

Reply:
Should we completely remove the GOE from the python API also?

Reply:
Yes, we should remove it.

But I propose to make it in the separate PR.

Reply:
Thanks for your comments, so what is the legit way to avoid `GetOutpuElement` in this case?

Reply:
You can use OutputVector (it contains ports) instead of NodeVector (it works with nodes).

Reply:
I replaced `GetOutputElement` with `lstmSequence->outputs().at(0)`.
What do you think?

Reply:
I see that `lstmSequence->output(x)` could be even better

Reply:
I think it is OK. But as I can see the method returns `NodeVector` what happens if we will use output with index  which is more that 0? Will it works?

Reply:
I mean return `NodeVector` with each of the outputs.
```
const auto Y = lstmSequence->output(0);
const auto Y_h = lstmSequence->output(1);
const auto Y_c = lstmSequence->output(2);

return {builder::opset1::reorder_axes(Y, {2, 1, 0, 3}),
        builder::opset1::reorder_axes(Y_h, {1, 0, 2}),
        builder::opset1::reorder_axes(Y_c, {1, 0, 2})};
```
The [ONNX importer LSTM tests](https://github.com/openvinotoolkit/openvino/blob/45899aab656c68341116121e353892c46d759456/ngraph/test/onnx/onnx_import_rnn.in.cpp) and the [type_prop tests](https://github.com/openvinotoolkit/openvino/blob/45899aab656c68341116121e353892c46d759456/ngraph/test/type_prop/lstm_sequence.cpp) check that.

Reply:
I've pushed commits with the discussed changes, could you rereview?